docs/contrib/CI.md

```diff
@@ -44,35 +44,6 @@ navigate Jenkins to access these features are available
     -   Target: all branches. Note that PRs will be run on public side after
         "Ready to Sync Public" workflow.
 
-
-## OpenROAD Flow
-
--   OpenROAD-flow-script-Public [folder]
-    -   `public_tests_all`
-        -   Description: runs flow tests except RTLMP designs. Should finish in
-            less than three hours.
-        -   Target: master branch.
-    -   `public_tests_all-pr`
-        -   Description: runs flow tests except RTLMP designs. Should finish in
-            less than three hours.
-        -   Target: all open PRs.
-    -   `publish-results-to-dashboard`
-        -   Description: uploads metrics to dashboard website.
-        -   Target: master branch.
--   OpenROAD-flow-scripts-Nightly-Public
-    -   Description: runs all flow tests including RTLMP designs.
-    -   Target: master branch.
--   OpenROAD-flow-scripts-Private [folder]
-    -   `public_tests_small`
-        -   Description: runs fast flow tests, does not include RTLMP designs.
-            Should finish in less than one hour.
-        -   Target: all branches that are not filed as PR. CI will run for PR
-            branches on the public side after "Ready to Sync Public" workflow.
--   OpenROAD-flow-scripts-All-Tests-Private
-    -   Description: runs flow tests, does not include RTLMP designs.
-    -   Target: secure branches.
```

Comment:
Let's replace with a link.

Reply:
I assumed that the documentation link for ORFS will be this: https://openroadflow.readthedocs.io. Is this a correct assumption?

Reply:
It should be https://openroad-flow-scripts.readthedocs.io/en/latest/